src/transformers/models/auto/tokenization_auto.py

```diff
@@ -107,7 +107,13 @@
             ),
             ("marian", ("MarianTokenizer" if is_sentencepiece_available() else None, None)),
             ("blenderbot-small", ("BlenderbotSmallTokenizer", None)),
-            ("blenderbot", ("BlenderbotTokenizer", None)),
+            (
+                "blenderbot",
+                (
+                    "BlenderbotTokenizer" if is_sentencepiece_available() else None,
```

Comment:
`BlenderbotTokenizer` does not need `sentencepiece`, so this shouldn't be changed.